### ISSUE TITLE
problem with overfitting

### ISSUE THREAD

**Issue by noorshev** [yongxuUSTC/sednn#11]
Hello Yong,
I am very much impressed with work. I am trying to use the code  , knowing that i split training set on many phases by saving model and weight and reload it each time for retraining (cause i had issue with machine performance (cpu and limited memory 8gb  ) , and i 'm facing this problem  the  testing loss increases while training accuracy is still improving . also i get wav file  audible but distorted.
i wish if you could upload the trained model (weight) Please . thanks very much .

Regards,
noor
